.github/workflows/ci.yml

```diff
@@ -135,3 +135,22 @@ jobs:
         uses: coverallsapp/github-action@master
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  build_e2e_test:
+      needs: [changes, install]
+      runs-on: ubuntu-latest
+      env:
+        NODE_OPTIONS: '--max-old-space-size=4096'
+      steps:
+        - uses: actions/checkout@v2
+        - name: Use Node.js
+          uses: actions/setup-node@v3
+          with:
+            node-version: 14
+            cache: 'npm'
+            cache-dependency-path: '**/package-lock.json'
+        - run: npm ci
+        - run: npm run build
```

Comment:
we should actually shard out the build into it's own step right? since they're common between both backend and e2e, then they can share on the build

Reply:
yarh

Reply:
yeah context is this is what we had initially, but the files from the build step weren't being used in e2e and backend tests so we combined the build, e2e and backend steps tgt. discussed w @karrui - can use a build artifact